packages/cli/src/commands/certs/ls.ts

```diff
@@ -38,10 +40,21 @@ async function ls(
     return 1;
   }
 
+  if (
+    typeof nextTimestamp !== undefined &&
+    typeof limit === 'number' &&
+    (Number.isNaN(limit) || limit > 100)
+  ) {
+    output.error('Please provide a number up to 100 for flag --limit');
+    return 1;
+  }
+
   // Get the list of certificates
-  const { certs, pagination } = await getCerts(client, nextTimestamp).catch(
-    err => err
-  );
+  const { certs, pagination } = await getCerts(
+    client,
+    nextTimestamp,
+    limit
+  ).catch(err => err);
```

Comment:
**suggestion (non-blocking):** remove `.catch(err => err)`

I know this wasn't changed in the PR, but while we're here, can re remove this exception handling `.catch(err => err)`? If `getCerts` throws, it looks like the result will be `certs` and `pagination` will be undefined, because those don't exist on an `Error` instance. Then we'll get a failure below when accessing `certs.length`, obfuscating the root error.

This also updates the types for `certs` and `pagination` here to be their true types instead of `any`, giving us more type protection.

Reply:
Would you mind if I filed an issue for that and did that in a seperate PR? This one has been dragging much longer than I originally hoped and smaller focused PRs are easier to write and merge.

Reply:
Certainly!